### ISSUE TITLE
🐛 Dropdown contents inaccessible by keyboard

### ISSUE THREAD

**Issue by tatianamac** [COVID19Tracking/website#865]
## Element
[US Current and Historical Data](https://covidtracking.com/api#status) (Note this is technically the link to the status above, but this element does not have an anchor link)

<img width="1182" alt="Dropdown activated with several GET button-looking things that are not tabable" src="https://user-images.githubusercontent.com/7484549/82085521-0f131a80-96a2-11ea-93c3-1c5882dcc190.png">

## Expected Behaviour
The ability to tab to the GET elements by keyboard.

## Actual Behaviour
Each of these GET elements is not tabbable. I can only access them by navigating by mouse.

## Recommendation
Make all get elements buttons if possible. If it can be a button, that would be best, as it'll bake in all the button goodness.

If it must be a div for some reason, it needs to inherit a role="button" and also get a lot of aria properties to make it function like a button. 

## Resources
[MDN's button role notes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).

**Comment by kevee** [comment]
@friej715 can we see if this is fixable via Swagger's UI?

**Comment by kevee** [comment]
I opened this issue with the Swagger-UI project:
https://github.com/swagger-api/swagger-ui/issues/6015

**Comment by friej715** [comment]
I'm going to take a look and see if I can propose a PR...

**Comment by friej715** [comment]
Actually, are we sure this is a Swagger-UI issue? I checked out that repo and ran the sample server, and it seems tabbable. I just don't know what would cause that to change here.

**Comment by kevee** [comment]
Closing in favor of #935
